release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -52,6 +52,11 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-release-notes-auth_{context}"]
 === Authentication and authorization
 
+[id="ocp-4-19-auth-ServiceAccountTokenNodeBinding_{context}"]
+==== Enable ServiceAccountTokenNodeBinding Kubernetes feature by default
+
+In {product-title} {product-version}, the `ServiceAccountTokenNodeBinding` feature is now enabled by default, aligning with upstream Kubernetes behavior. This feature allows service account tokens to be bound directly to Node objects in addition to the existing binding options. Benefits of this change include enhanced security through automatic token invalidation when bound nodes are deleted, better protection against token replay attacks across different nodes, and improved audit capabilities for tracking token usage by specific infrastructure.
```

Comment:
> , and improved audit capabilities for tracking token usage by specific infrastructure

To speak strictly, if this third benefit about tracking token usage is regarding the bound serviceaccount tokens' audit logs (including the annotation `authentication.k8s.io/issued-credential-id` for the token issuing and the annotation `authentication.k8s.io/credential-id` for the token usage), then it is the benefit of another K8S feature `ServiceAccountTokenJTI` (which is already enabled in a previous release before 4.19), instead of `ServiceAccountTokenNodeBinding`. CC @ibihim 

Here is just a minor nit from stricter speaking perspective. The PR is still LGTM to keep above wording which is not that strict :)